CTP/common/src/com/navercorp/cubridqa/common/ConfigParameterConstants.java

```diff
@@ -44,6 +45,7 @@
 	public static final String ENABLE_STATUS_TRACE_YES_OR_NO = "enable_status_trace_yn";
 	public static final String ENABLE_SAVE_LOG_ONCE_FAIL_YES_OR_NO = "enable_save_log_once_fail_yn";
 	public static final String DELETE_TESTCASE_AFTER_EACH_EXECUTION_YES_OR_NO = "delete_testcase_after_each_execution_yn";
+	public static final String CLEAN_PROCESS_AFTER_EACH_EXECUTION_YES_OR_NO = "clean_processes_after_each_execution_yn";
```

Comment:
There are two with 'each_execution_yn' suffix but they are different.
delete_testcase_after_each_execution_yn
clean_processes_after_each_execution_yn 

The first one will delete test case after each test case has been done.
The second will clean all processes after whole test has been completed.

So, it's better to change the second one to another name.